File: 283_Move_Zeroes/main.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        lastZeroIndex = 0
        for i, _ in enumerate(nums):
            if nums[i] != 0:
                nums[i], nums[lastZeroIndex] = nums[lastZeroIndex], nums[i]
                lastZeroIndex += 1

File: 283_Move_Zeroes/main.go
package main

func moveZeroes(nums []int) {
	lastZeroIndex := 0

	for i := range nums {
		if nums[i] != 0 {
			nums[i], nums[lastZeroIndex] = nums[lastZeroIndex], nums[i]
			lastZeroIndex++
		}
	}
}
